tests/core/examples/evolution_examples/1/output_manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+   "metadata":{
+      "pipeline_name":"test_pipeline",
+      "base_path":"gs://bucket",
+      "run_id":"custom_run_id",
+      "component_id":"example_component"
+   },
+   "index":{
+      "location":"/example_component"
+   },
+   "fields": {
+      "images_width": {
+         "type": "int32",
+         "location":"/example_component"
+      },
+      "images_height": {
+         "type": "int32",
+         "location":"/example_component"
+      },
+      "images_data": {
+         "type": "binary",
+         "location":"/example_component"
+      },
+      "captions_data": {
+         "type": "binary",
+         "location":"/example_component"
+      },
+      "embeddings_data": {
+         "type": "list<item: float>",
```

Comment:
I don't think that's the intended way of storing the types in the output manifest. They should match the format defined in the component spec: 
```bash
 "type": array
  "items":
    "type": float32
```